jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/saved/SavedFiltersPresenter.java

```diff
@@ -69,8 +69,12 @@ public void setFilterSettingsManager(final FilterSettingsManager filterSettingsM
         this.filterSettingsManager = filterSettingsManager;
     }
 
-    protected void loadSavedFilters() {
-        filterSettingsManager.loadSavedFilters(filters -> filters.forEach(f -> view.addSavedFilter(f)));
+    protected void loadSavedFilters(final String defaultFilter) {
+        view.removeAllSavedFilters();
+        filterSettingsManager.loadSavedFilters(filters -> filters.forEach(f -> {
+            f.setDefaultFilter(f.getKey().equals(defaultFilter));
```

Comment:
see the previous comment, this probably should be done at the filter manager level for better encapsulation.